post.js

```diff
@@ -0,0 +1,163 @@
+async function loadPost(url) {
+  try {
+    const response = await fetch(url);
```

Comment:
We should have an Accept header for the json and activitypub mime types as well as text/html. That way servers that support serving AP according to mime type will let us skip the HTML parsing step.